src/Symfony/Component/Validator/Constraints/BackedEnumValue.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * Validates that a value is one of an enum.
```

Comment:
```suggestion
 * Validates that a backed enum can be hydrated from a value.
```

We should add a piece of documentation here, that directs people to `Type` and `Choice` constraints, if they want to validate actual enums instead the backing values. The whole discussion in #43047 shows that people mix this up regularly.